### PR TITLE
#372 cross accounts role

### PR DESCRIPTION
**Pull request by mrandi**
partially fix for #372

### CODE REVIEW

File: fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V12__add_cross_account_role_type.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  id,
+  help_text,
+  violation_severity,
+  priority,
+  title,
+  created,
+  created_by,
+  version)
+VALUES ('CROSS_ACCOUNT_ROLE',
+        '',
```
Comment (mrandi) [inline fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V12__add_cross_account_role_type.sql]: need some suggestions for the help text and the severity!

File: fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/policy/CrossAccountPolicyForIAMJob.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                .withRegion(region)
+                .withPluginFullyQualifiedClassName(CrossAccountPolicyForIAMJob.class)
+                .withType(CROSS_ACCOUNT_ROLE)
+                .withMetaInfo(metaInfo)
+                .withInstanceId(roleId)
+                .withEventId(EVENT_ID).build();
+        violationSink.put(violation);
+    }
+
+}
```
Comment (Gregsen) [inline fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/policy/CrossAccountPolicyForIAMJob.java]: please add tests :)

**Comment by Gregsen** [comment]
👍

**Comment by harti2006** [comment]
:+1:

File: fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V12__add_cross_account_role_type.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  id,
+  help_text,
+  violation_severity,
+  priority,
+  title,
+  created,
+  created_by,
+  version)
+VALUES ('CROSS_ACCOUNT_ROLE',
+        'Your account has an IAM policy that allows access from another account. Cross account access is should be avoided',
```
Comment (harti2006) [inline fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V12__add_cross_account_role_type.sql]: typo

**Pull request merged**